### DISCUSSION TITLE
[Now Resolved] GitHub Graduation - Not Recognising GitHub Student

### DISCUSSION THREAD

**Discussion by lornaarmstrong** [community/community#17794] [GitHub Education]
I study a four year course, and I'm due to graduate this year. I tried to join the GitHub Graduation, however the autochecker for my Pull Request says: I'm not seeing a valid student developer pack approval, please submit an [application](https://education.github.com/discount_requests/student_application?utm_source=2021-06-05-GitHubGraduation) to get started

However, I have already applied, and when I click on the application, it shows this:
![image](https://user-images.githubusercontent.com/43929997/166666679-9deb40af-1e1b-440f-b621-508e34a6583a.png)

What am I doing wrong?

**Comment by juanpflores** [comment]
Thanks for opening this question. the bot had a couple of rough hours but it should be working again. Glad to read this issue was resolved and congratulations on graduating 🎓
